mailpoet/assets/js/src/common/loader/loading_spinner.tsx

```diff
@@ -0,0 +1,14 @@
+type LoadingSpinnerProps = {
+  alt: string;
+  className?: string;
+};
+
+export function LoadingSpinner({ alt, ...restProps }: LoadingSpinnerProps) {
```

Comment:
Thank you for calling this out. tldr I'm going to update this to use the WP components [Spinner](https://wordpress.github.io/gutenberg/?path=/docs/components-spinner--docs), which we're already using elsewhere in the codebase, unless you happen to have a better suggestion.

It's not part of the specs, but the method we do have to indicate loading on the send step blurs out the entire form and prevents editing. I was thinking that, especially if a user has very large/complex lists that take some time to calculate, it would be very annoying to have to wait until the API request completes before making other changes.

The spinner image is something we already use in a few other places ([example1](https://github.com/mailpoet/mailpoet/blob/cf1e89d39f285465177c4a5db32b7f848f6afabc/mailpoet/assets/css/src/generic/_helpers.scss#L86-L86), [example2](https://github.com/mailpoet/mailpoet/blob/cf1e89d39f285465177c4a5db32b7f848f6afabc/mailpoet/assets/css/src/generic/_buttons-spinners.scss#L2-L2)), but those didn't seem to be easily reusable and I thought having a component might be nice for the future.

Reply:
> tldr I'm going to update this to use the WP components [Spinner](https://wordpress.github.io/gutenberg/?path=/docs/components-spinner--docs)

Sounds like a good plan to me. Thanks.